src/lidar/read_rp_lidar.h

```diff
@@ -9,16 +9,16 @@ namespace lidar {
 
 bool initializeLidar();
 
-void readLidarLoop();
+DataPoint<navtypes::points_t> readLidar();
 
-typedef struct Polar2D {
+struct Polar2D {
 	double r, theta;
 	double asCartesianX() {
 		return r * cos(theta);
 	}
 	double asCartesianY() {
 		return r * sin(theta);
 	}
-} Polar2D;
+};
```

Comment:
Can you add comments for this struct too? Just minor stuff like degrees vs rads or something would be good.